pom.xml

```diff
@@ -1077,6 +1095,21 @@
         <artifactId>xpp3</artifactId>
         <version>${xpp3.version}</version>
       </dependency>
+      <dependency>
```

Comment:
Like for maven properties names, all dependencies are sorted by groupId name in alphabetical order.